src/Omnibox.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  omnibox: {
+    position: 'absolute',
+    margin: '10px',
+    width: '400px',
+  },
+});
+
+class MapOmnibox extends Component {
+  constructor(props) {
+    super(props);
+    this.onPlaceChanged = this.onPlaceChanged.bind(this);
+  }
+
+  onPlaceChanged() {
+    const place = this.autocomplete.getPlace();
+    const { lat, lng } = place.geometry.location;
+    this.props.map.setCenter({
+      lat: lat(),
+      lng: lng(),
+    });
+  }
+
+  render() {
+    if (!this.props.map || !this.props.maps) return null;
+
+    return (
+      <View style={styles.omnibox}>
```

Comment:
[![Screenshot from Gyazo](https://gyazo.com/d784d4ec1d6d5d516ed6ade24504ad17/raw)](https://gyazo.com/d784d4ec1d6d5d516ed6ade24504ad17)

I don't know how to apply `height` property for `<input>` .